cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/validator/AwsIDBrokerObjectStorageValidator.java

```diff
@@ -58,30 +63,32 @@ public ValidationResult validateObjectStorage(AmazonIdentityManagementClient iam
             if (instanceProfile != null) {
                 CloudIdentityType cloudIdentityType = cloudFileSystem.getCloudIdentityType();
                 if (CloudIdentityType.ID_BROKER.equals(cloudIdentityType)) {
-                    validateIDBroker(iam, instanceProfile, cloudFileSystem, resultBuilder);
+                    validateIDBroker(iam, instanceProfile, cloudFileSystem, backupLocationBase, resultBuilder);
```

Comment:
Can we have a log about what type of validation IDBroker or not and on what backupLocationBase it has been triggered?

Reply:
addressed.